packages/ds-medicare-gov/src/styles/components/_override.help_drawer.scss

```diff
@@ -2,31 +2,15 @@
 $mct-drawer-close-button-size: 26px;
 $mct-drawer-heading-font-size: $h1-mobile-font-size;
 
-.mct-c-help-drawer {
-  .ds-c-drawer__header {
-    background-color: $drawer-header__background-color;
-  }
-
-  .ds-c-drawer__header-heading {
-    font-family: $font-family-montserrat;
-    font-size: $mct-drawer-heading-font-size;
-    font-weight: $font-weight-bold;
-    text-align: left;
-  }
-
-  .ds-c-drawer__close-button {
```

Comment:
Relying on Button variation=ghost and font-size class set on icon to achieve this, so it could be removed.